website/docs/docs/dbt-versions/release-notes/04-Sept-2023/product-docs-summer-rn.md

```diff
@@ -0,0 +1,37 @@
+
+What a busy summer! We merged 256 PRs between July 1st and August 31. We’d like to thank the 22 community members who contributed for the 1st time to docs.getdbt.com :pray: Here's what's new to [docs.getdbt.com](http://docs.getdbt.com/):
+
+* Changes based on YOUR feedback (you can reach us most directly by opening a pull request or issue in [our repo](https://github.com/dbt-labs/docs.getdbt.com) or reaching out in the dbt community Slack channel #dbt-product-docs): 
+	- [Permissions table](https://docs.getdbt.com/docs/cloud/manage-access/enterprise-permissions) for Enterprise accounts
+	- [Browser session page](https://docs.getdbt.com/docs/cloud/about-cloud/browsers#browser-sessions) that clarifies dbt Cloud’s browser session time and when it logs users off.
+* The docs site now collapses sections you’re not currently looking at. There were quite a few people who wanted this, and it bugged us too, so we were happy to get this shipped!
+* Introduced the idea of [“Trusted” adapters](/docs/supported-data-platforms#types-of-adapters).
+
+## ☁ Cloud projects
+
+* What’s new? is back in the dbt Cloud UI! The Docs team will begin updating the content with new features.
+* Launched the re-released [Semantic Layer beta docs](/docs/use-dbt-semantic-layer/dbt-sl), which introduce users to the new API, new guide to set up MetricFlow and the new Semantic Layer, as well as revamp the ‘Use the dbt Semantic Layer’ section for users.
+* Updated [Admin API v2 and v3](/docs/dbt-cloud-apis/admin-cloud-api) to help you understand the differences between them and which version includes the endpoints you use.
+* To improve discoverability, the docs team made changes to the [deploy dbt sidebar](/docs/deploy/deployments). We added cards and aligned better with the dbt Cloud UI and the way it’s used.
+* Deprecated legacy job schemas in the [Discovery API](/docs/dbt-cloud-apis/discovery-api).
+* Added a page to describe [experimental and beta features](/docs/dbt-versions/experimental-features) in dbt Cloud and what you need to know abotu them.
+
+## 🎯 Core projects
+
+* We released [dbt 1.6](/guides/migration/versions/upgrading-to-v1.6)! We added docs for the new commands `dbt retry` and `dbt clone`
+
+## New 📚 Guides, ✏️ blog posts, and FAQs
+* Check out how these community members use the dbt community in the [Community spotlight](/community/spotlight). 
+* Blog posts published this summer include [Optimizing Materialized Views with dbt](/blog/announcing-materialized-views),  [Data Vault 2.0 with dbt Cloud](/blog/data-vault-with-dbt-cloud), and [Create dbt Documentation and Tests 10x faster with ChatGPT](/blog/create-dbt-documentation-10x-faster-with-chatgpt) 
+* We now have two new best practice guides: [How we build our metrics](/guides/best-practices/how-we-build-our-metrics/semantic-layer-1-intro) and [Customizing CI/CD with Custom Pipelines](/guides/orchestration/custom-cicd-pipelines/1-cicd-background).
```

Comment:
the CI/CD one is actually here https://docs.getdbt.com/guides/orchestration/set-up-ci/overview and is a revamp of best practices from across devhub as opposed to brand-new. (The guide you linked to used to contain some of the content that is now in the set up CI block, so it probably did appear in your diff view.)

Reply:
Yep, the diff is where I got it from. Thanks so much for keeping me honest, Joel!